db/migrations/versions/~2024_03_21_1411-7b877df0bd36_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 7b877df0bd36
+Revises: 2fd177385a5c
+Create Date: 2024-03-21 14:11:56.510506
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7b877df0bd36"
+down_revision = "2fd177385a5c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("round", schema=None) as batch_op:
+        batch_op.alter_column("application_reminder_sent", existing_type=sa.BOOLEAN(), nullable=False)
```

Comment:
If this column `application_reminder_sent` is altered. Then there might be errors in migration if existing data is `null` ?

Reply:
Is it null anywhere? I checked my local and test db and they're all filled in, so figured it would be okay

Reply:
ok. then should be fine in that case.